weave/scorers/llm_scorer.py

```diff
@@ -14,57 +9,26 @@ class LLMScorer(weave.Scorer):
     This scorer leverages LLMs to evaluate and score model outputs. It provides a flexible
     way to use different LLM providers for scoring purposes.
 
-    Attributes:
-        client: An instantiated LLM client with valid API credentials
-        model_id: The specific model identifier to use for scoring
-    """
-
-    client: _LLM_CLIENTS = Field(
-        description="The LLM client to use, has to be instantiated with an api_key"
-    )
-    model_id: str = Field(description="The model to use")
-
-    @field_validator("client")
-    def validate_client(cls, v: _LLM_CLIENTS) -> _LLM_CLIENTS:
-        client_type_name = type(v).__name__
-        if client_type_name not in _LLM_CLIENTS_NAMES:
-            raise ValueError(
-                f"Invalid client type. Expected one of {_LLM_CLIENTS_NAMES}, got {client_type_name}"
-            )
-        return v
-
-
-class InstructorLLMScorer(weave.Scorer):
```

Comment:
It may be helpful to keep this class, but override `__new__` to say that the class is deprecated and what to use instead